ccm_web/client/src/components/MultipleUserEnrollmentWorkflow.tsx

```diff
@@ -132,8 +135,12 @@ export default function MultipleUserEnrollmentWorkflow (props: MultipleUserEnrol
     if (props.getSectionsError !== undefined) return getSectionsErrorAlert
 
     const canCreate = isAuthorizedForRoles(props.userCourseRoles, createSectionRoles)
+    const onSectionCreated = (section: CanvasCourseSection): void => {
+      setSelectedSection(injectCourseName([section], props.course.name)[0])
+      props.onSectionCreated(section)
+    }
```

Comment:
I realized when working on `AddUMUsers.tsx` that I had forgotten to make sure the new section was selected here.